pyproblems/derivative_x.py

```diff
@@ -0,0 +1,21 @@
+'''This program gives the derivative of "x" raised to the nth power,
+where n ranges from "-infinity to +infinity"'''
+def derivative_x_pow(dif, num):
+    '''The input should be integers, the function will give return the
+    nth derivative of "x" raised to the power of the given number'''
+    if not isinstance(num, int) or isinstance(num, bool):
+        raise TypeError("unsupported format")
+    if not isinstance(dif, int) or isinstance(dif, bool):
+        raise TypeError("unsupported format")
+    if dif <= 0:
+        raise ValueError("invalid entry, enter a valid number to differentiate")
```

Comment:
Enter? User is not entering any value but passing values. So change text accordingly